data/human/deep missions.txt

```diff
@@ -2286,6 +2287,7 @@ mission "Deep: Scientist Rescue 3A"
 		set "license: City-Ship"
 		event "battle in arneb end"
 		log `Destroyed the pirate Bactrian that was attempting to prevent the rescue of the Star Queen and was given a city-ship license by Paris.`
+		log "People" "Beelzebub" `A pirate Bactrian you were responsible for destroying may have been captained by the mysterious Beelzebub whom you have thwarted once before. The Bactrian was destroyed, but you received a message afterwards suggesting they may still not be gone for good, if it was even truly them...`
```

Comment:
Normal log entries avoid the use of first and second person pronouns; I'm unsure if the same is the case for special log entries, but we should look into that.

Reply:
> Since you are adding logs to the hai mission file - IMO Nanachi should have an entry, too.

Feel free to write an entry and I can put it in. Most of the logs came out of a request, I didn't fabricate them.

Reply:
People/faction logs all appear to speak about the person/faction in question without referencing the player, so if we want to talk about something the player did, then that should be a normal log (and not use "you" or "I").

Reply:
I really want to express that it's unclear if this was actually Beelzebub though while still attaching it to the People log for Beelzebub. Do you have any suggestions how I might go about that?